host_vars/127.0.0.1

```diff
@@ -50,6 +50,7 @@ charon_base_url: http://nestor1:4242
 # FIXME: hg19 was empty though. Needs to fetch from somewhere else. 
```

Comment:
This FIXME I wrote when I was unsure if we would need hg19, but I'm now guessing we don't, right?